bot.py

```diff
@@ -641,9 +642,44 @@ def respond_to_ig_link_with_instafix(original_message, link):
 
 
 @bot.message_handler(regexp="http", func=lambda message: message.from_user.id in ALLOWED_USERS or message.chat.id in ALLOWED_CHATS)
-def handle_link(message):
-    if message.chat.id not in ALLOWED_CHATS:
-        bot.reply_to(message, "This site is not supported yet\.")
+def handle_unknown_link(message):
```

Comment:
it has to be rewritten. No code related to mastodon should be in bot.py
I would see it in something like that
```
def handle_unknown_link(msg):
    array_of_links = mastodon_handler.extract_links(messageText)
    for link in array_of_links:
        if mastodon_handler.is_mastodon_link(link):
            handler_response = maston_handler.handle_url(link)
            send_post_to_tg(handler_response)
```